### PR TITLE
LPS-142626

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina, @tinatian

This is resubmitted following your comments in #2465
instead of returning null in _listClassPathFiles(), it continues to the next iteration of classNames.
if filelist comes out to be empty before we set sb and boostrapSB, log the error.

Thank you!

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java [hunk 1]
```diff
@@ -189,142 +190,178 @@ private static String _buildClassPath(
...
 		}
 
-		String protocol = url.getProtocol();
+		return false;
+	}
 
-		if (protocol.equals("bundle") || protocol.equals("bundleresource")) {
-			try {
-				URLConnection urlConnection = url.openConnection();
+	private static List<File> _listClassPathFiles(
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PortalClassPathUtil.java]: @julschong Do you have a better solution to fix the issue I mentioned without modifying _listClassPathFiles? Please slack me tomorrow and we can have a talk.

Tina.

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by julschong** [comment]
ci:test

**Pull request closed without merging**